src/main/java/com/conveyal/analysis/controllers/RegionalAnalysisController.java

```diff
@@ -354,11 +355,12 @@ private Object getCsvResults (Request req, Response res) {
             throw AnalysisServerException.notFound("Path results were not recorded for this analysis");
         }
 
-        String key = regionalAnalysisId + resultType + ".csv.gz";
+        // TODO result path is stored in model, use that?
+        String key = regionalAnalysisId + '_' + resultType + ".csv.gz";
         FileStorageKey fileStorageKey = new FileStorageKey(config.resultsBucket(), key);
-        JSONObject json = new JSONObject();
-        json.put("url", fileStorage.getURL(fileStorageKey));
-        return json.toJSONString();
+
+        res.type("text");
```

Comment:
Doesn't the type need to be a MIME type?

Used here: https://github.com/conveyal/analysis-ui/commit/f714b49b74f5b08792ea3c8f8896cd88e8c0a478#diff-b19858859df3fd3e2d21ecd70a5d70a41995b47fa337915004a99b7eba4d39faR122-R127